applications/API/tests/test_forms.py

```
@@ -0,0 +1,288 @@
+        form_data.update(
+            {
+                "insult_id": 999,  # Non-existent ID
+            }
+        )
+        form = InsultReviewForm(data=form_data)
+
+        with self.assertRaises(ValidationError) as context:
+            form.clean()
+
+        expected_message = "Invaild Insult ID - Please confirm Insult ID"
+        self.assertEqual(str(context.exception), expected_message)
```

Ruff (0.12.2)

Use a regular `assert` instead of unittest-style `assertEqual`

Replace `assertEqual(...)` with `assert ...`

(PT009)